qa/tasks/cephfs/test_fscrypt.py

```diff
@@ -90,6 +91,273 @@ def verify_alternate_name():
 
         self.mount_a.run_shell_payload(f"cd {self.path} && stat {file}")
 
+class TestFSCryptRMW(FSCryptTestCase):
+    CLIENTS_REQUIRED = 2
+    def setUp(self):
+        super().setUp()
+        self.mount_b.run_shell_payload(f"sudo fscrypt unlock --quiet --key=/tmp/key {self.path}")
```

Comment:
so these ioctls work now?

Reply:
Yep we are using a custom fscrypt cli now.